src/utils/color-utils.ts

```diff
@@ -38,7 +41,7 @@ export function hexToRgb(hex) {
   return [r, g, b];
 }
 
-export function isHexColor(hex) {
+export function isHexColor(hex: string) {
```

Comment:
Use type guard here `isHexColor(hex:string): hex is HexColor`

Reply:
this function works differently, it returns a RegExpExecArray used to get the RGB component of the hex color. 
Should I refactor this?

Reply:
oh okay, nvm. disregard my comment. this should `isHexColor(hex: string): RGBColor | null`